Add CONTRIBUTING.md and CODE_OF_CONDUCT.md (#16)
Archish27/nora-covid-19-bot: CONTRIBUTING.md

<!-- omit in toc -->

> # Contributing to Nora Bot - Covid-19

Reviewer: ```suggestion
# Contributing to Nora Covid-19 Bot
```

> And if you like the project, but just don't have time to contribute, that's fine. There are other easy ways to support the project and show your appreciation, which we would also be very happy about:
> - Star the project
> - Tweet about it
> - Refer this project in your project's readme
> - Mention the project at local meetups and tell your friends/colleagues

<!-- omit in toc -->
## Table of Contents

- [Code of Conduct](#code-of-conduct)
- [I Have a Question](#i-have-a-question)
- [I Want To Contribute](#i-want-to-contribute)
- [Reporting Bugs](#reporting-bugs)
- [Suggesting Enhancements](#suggesting-enhancements)
- [Your First Code Contribution](#your-first-code-contribution)
- [Improving The Documentation](#improving-the-documentation)
- [Styleguides](#styleguides)
- [Commit Messages](#commit-messages)
- [Join The Project Team](#join-the-project-team)

## Code of Conduct

This project and everyone participating in it is governed by the

> [Nora Bot - Covid-19 Code of Conduct](https://github.com/Archish27/nora-covid-19-botblob/master/CODE_OF_CONDUCT.md).

Reviewer: ```suggestion
[Nora Covid-19 Bot Code of Conduct](https://github.com/Archish27/nora-covid-19-bot/blob/master/CODE_OF_CONDUCT.md).
```

By participating, you are expected to uphold this code. Please report unacceptable behavior.

## I Have a Question

> Before you ask a question, it is best to search for existing [Issues](https://github.com/Archish27/nora-covid-19-botissues) that might help you. In case you have found a suitable issue and still need clarification, you can write your question in this issue. It is also advisable to search the internet for answers first.

Reviewer: ```suggestion
Before you ask a question, it is best to search for existing [Issues](https://github.com/Archish27/nora-covid-19-bot/issues) that might help you. In case you have found a suitable issue and still need clarification, you can write your question in this issue. It is also advisable to search the internet for answers first.
```

If you then still feel the need to ask a question and need clarification, we recommend the following:

> - Open an [Issue](https://github.com/Archish27/nora-covid-19-botissues/new).

Reviewer: ```suggestion
- Open an [Issue](https://github.com/Archish27/nora-covid-19-bot/issues/new).
```

- Provide as much context as you can about what you're running into.
- Provide project and platform versions (python, Visual C++, OS etc), depending on what seems relevant.

We will then take care of the issue as soon as possible.

## I Want To Contribute

To contribute via pull request, follow these steps:

> 1. Create an issue describing the feature you want to work on (or
>    have a look at the [issues](https://github.com/browserstack/ws-reconnect-proxy/issues))

Reviewer: ```suggestion
1. Create an issue describing the feature you want to work on or have a look at the [issues(https://github.com/Archish27/nora-covid-19-bot/issues)
```

2. Write your code
3. Create a pull request describing your change

Your pull request will be reviewed by @Archish27, will get back to you about any necessary changes or questions.

### Reporting Bugs

<!-- omit in toc -->
#### Before Submitting a Bug Report

A good bug report shouldn't leave others needing to chase you up for more information. Therefore, we ask you to investigate carefully, collect information and describe the issue in detail in your report. Please complete the following steps in advance to help us fix any potential bug as fast as possible.

- Make sure that you are using the latest version.
- Determine if your bug is really a bug and not an error on your side e.g. using incompatible environment components/versions (Make sure that you have read the [documentation](). If you are looking for support, you might want to check [this section](#i-have-a-question)).

> - To see if other users have experienced (and potentially already solved) the same issue you are having, check if there is not already a bug report existing for your bug or error in the [bug tracker](https://github.com/Archish27/nora-covid-19-botissues?q=label%3Abug).

Reviewer: ```suggestion
- To see if other users have experienced (and potentially already solved) the same issue you are having, check if there is not already a bug report existing for your bug or error in the [bug tracker](https://github.com/Archish27/nora-covid-19-bot/issues?q=label%3Abug).
```

- Also make sure to search the internet (including Stack Overflow) to see if users outside of the GitHub community have discussed the issue.
- Collect information about the bug:
- Stack trace (Traceback)
- OS, Platform and Version (Windows, Linux, macOS, x86, ARM)
- Version of the interpreter, compiler, SDK, runtime environment, package manager, depending on what seems relevant.
- Possibly your input and the output
- Can you reliably reproduce the issue? And can you also reproduce it with older versions?

<!-- omit in toc -->
#### How Do I Submit a Good Bug Report?

> You must never report security related issues, vulnerabilities or bugs to the issue tracker, or elsewhere in public. Instead sensitive bugs must be sent by email to <>.
<!-- You may add a PGP key to allow the messages to be sent encrypted as well. -->

We use GitHub issues to track bugs and errors. If you run into an issue with the project:

> - Open an [Issue](https://github.com/Archish27/nora-covid-19-botissues/new). (Since we can't be sure at this point whether it is a bug or not, we ask you not to talk about a bug yet and not to label the issue.)

Reviewer: ```suggestion
- Open an [Issue](https://github.com/Archish27/nora-covid-19-bot/issues/new). (Since we can't be sure at this point whether it is a bug or not, we ask you not to talk about a bug yet and not to label the issue.)
```

- If the team is able to reproduce the issue, it will be marked `needs-fix`, as well as possibly other tags (such as `critical`), and the issue will be left to be [implemented by someone](#your-first-code-contribution).

<!-- You might want to create an issue template for bugs and errors that can be used as a guide and that defines the structure of the information to be included. If you do so, reference it here in the description. -->

### Suggesting Enhancements

This section guides you through submitting an enhancement suggestion for Nora Bot - Covid-19, **including completely new features and minor improvements to existing functionality**. Following these guidelines will help maintainers and the community to understand your suggestion and find related suggestions.

<!-- omit in toc -->
#### Before Submitting an Enhancement

- Make sure that you are using the latest version.
- Read the [documentation](https://github.com/Archish27/nora-covid-19-bot/blob/master/README.md) carefully and find out if the functionality is already covered, maybe by an individual configuration.

> - Perform a [search](https://github.com/Archish27/nora-covid-19-botissues) to see if the enhancement has already been suggested. If it has, add a comment to the existing issue instead of opening a new one.

Reviewer: ```suggestion
- Perform a [search](https://github.com/Archish27/nora-covid-19-bot/issues) to see if the enhancement has already been suggested. If it has, add a comment to the existing issue instead of opening a new one.
```

- Find out whether your idea fits with the scope and aims of the project. It's up to you to make a strong case to convince the project's developers of the merits of this feature. Keep in mind that we want features that will be useful to the majority of our users and not just a small subset. If you're just targeting a minority of users, consider writing an add-on/plugin library.

<!-- omit in toc -->
#### How Do I Submit a Good Enhancement Suggestion?

> Enhancement suggestions are tracked as [GitHub issues](https://github.com/Archish27/nora-covid-19-botissues).

Reviewer: ```suggestion
Enhancement suggestions are tracked as [GitHub issues](https://github.com/Archish27/nora-covid-19-bot/issues).
```
